diesel/src/macros/mod.rs

```diff
@@ -833,8 +845,12 @@ mod tests {
         use macros::tests::my_types::*;
 
         /// Table documentation
-        table_with_custom_type_and_id (a){
+        ///
+        /// some in detail documentation
```

Comment:
Yeah, either running rustdoc and checking the html, or running something like cargo-expand and comparing its output.

I'm actually a bit surprised I couldn't find a crate for testing macros. Something like piping `cargo rustc -- --pretty=expanded` into syn (at least its tokenizer) and comparing that to some `quote!`ed expressions. That may be an interesting project in itself.